src/lib/controller/user-controller.ts

```diff
@@ -28,7 +28,34 @@ export class UserController implements IUserController {
     }
 
     async getProjects(): Promise<Project[]> {
-        throw new Error("Method not implemented.");
+        const response = await this.client.get("projects");
+        try {
+            const projects: Project[] = await response.json();
+            return Promise.resolve(projects);
+        } catch (error: unknown) {
+            return Promise.reject(error);
+        }
```

Comment:
Now that you added that noop-catch this isn't required. I tested this in different scenarios, e.g., when an error is thrown in this method and if the response is HTML and every time the error is caught and `Error: Could not load projects.` is displayed.

So this can just be
```suggestion
        return await this.client.get("projects").then((response) => response.json());
```

This applies to the other methods too.

Reply:
Thank you, forgot this.

Reply:
Hopefully, I did not introduce a new error, as I cleaned this up.

Reply:
Is there a reason why you didn't do that in `getAllProjects`?

Reply:
Yes, because it forgot it ;)